src/components/welcome/Welcome.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './welcome.css';
+
+function Welcome() {
+  return (
+  <section className="welcome">
+    <span className="welcome__text">Sticky Header!</span>
+  </section>
+  );
+}
+
+export default Welcome;
```

Comment:
Зроби порожню строку на кінці файлу
```suggestion
export default Welcome;

```